### PR TITLE
Feature/flipper

### PR DESCRIPTION
**Pull request by hilarious23**

### CODE REVIEW

File: docs/builder-guides/from-zero-to-ink-hero/flipper-contract/flipper-contract.md [hunk 1]
```diff
@@ -1,28 +1,130 @@
...
+in this tutorial we will compile&deploy flipper contract written in ink! and interact with it from flipper application UI.
 
-### What will we do?
-In this tutorial we will implement the most basic contract: [Flipper](https://github.com/paritytech/ink/blob/v3.3.0/examples/flipper/lib.rs) in ink!.
+## What will we use ?
 
-### What will we use?
-- [ink! 3.3.0](https://github.com/paritytech/ink/tree/v3.3.0)
+[ink! 3.3.0](https://github.com/paritytech/ink/tree/v3.3.0)   
+[flipper(wasm-showcase-dapps)](https://github.com/AstarNetwork/wasm-showcase-dapps/tree/main/flipper)
```
Comment (bobo-k2) [inline docs/builder-guides/from-zero-to-ink-hero/flipper-contract/flipper-contract.md:24]: Broken link

File: docs/builder-guides/from-zero-to-ink-hero/flipper-contract/flipper-contract.md [hunk 2]
```diff
@@ -1,28 +1,130 @@
...
+- Unit test your contract
+- Detecting Polkadot.js
+- Getting strorage date from ink! contracts
+- Executing functions of ink! contracts
+- Watching the status of transactions and getting their results
+- Structure of repo of web application to interact with contracts from UI
+
+## Summary
+
+[I. File & Folder structure of the project](./Structure/file-structure.md)
```
Comment (bobo-k2) [inline docs/builder-guides/from-zero-to-ink-hero/flipper-contract/flipper-contract.md:41]: Looks like this one is broken too

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: docs/builder-guides/from-zero-to-ink-hero/flipper-contract/flipper-contract.md [hunk 3]
```diff
@@ -1,28 +1,130 @@
...
+### Setting
+
+Clone repo 
+
+```bash
+git clone https://github.com/AstarNetwork/wasm-showcase-dapps
+```
+and Install dependencies by running `yarn`
+```bash
+cd wasm-showcase-dapps/flipper
```
Comment (bobo-k2) [inline docs/builder-guides/from-zero-to-ink-hero/flipper-contract/flipper-contract.md:60]: I can't find `flipper` folder in `wasm-showcase-dapps`

**Comment by hilarious23** [comment]
I removed flipper from wasm-showcase and create dedicated repo.
https://github.com/AstarNetwork/wasm-flipper

**Comment by fiexer** [comment]
Any updates on the changes @hilarious23 ?

**Review by andabak — CHANGES_REQUESTED** [review]
@hilarious23 can you rebase this to main branch (big changes after PR253 merged) - new Build chapter.

cc: @bobo-k2 @niklabh

**Comment by Maar-io** [comment]
This PR will be replaced with the updates in Flipper tutorial for for ink4! 
@ashutoshvarma please consider these changes in your PR for Flipper ink4, and then cancel this PR.

**Pull request closed without merging**